source/install/docker-local-machine.rst

```diff
@@ -95,27 +95,10 @@ Fedora
        
 3. When Docker is done fetching the image, open http://localhost:8065/ in your browser.
 
-Arch
-^^^^
-
-1. Install Docker using the following commands:
-
-   .. code:: bash
-
-       pacman -S docker
-       systemctl enable docker.service
-       systemctl start docker.service
-       gpasswd -a <username> docker
-       newgrp docker
-
-2. Start Docker container:
-
-   .. code:: bash
-
-       docker run --name mattermost-preview -d --publish 8065:8065 mattermost/mattermost-preview
+Arch Linux
+^^^^^^^^^^
 
-3. When Docker is done fetching the image, open ``http://localhost:8065/``
-   in your browser.
+`Use the following online guide from the Arch Linux wiki<https://wiki.archlinux.org/index.php/Mattermost>`_.
```

Comment:
That's OK, but links inside the Arch Linux article are subject to change. Yesterday as I finished my changes against the Arch Linux documentation, a Trusted User changed everything to [make it more concise](https://wiki.archlinux.org/index.php?title=Mattermost&action=history).
So the idea is to avoid link ids like `#With_Docker` as much as possible.

Btw, what have you done to be able to publish your own code on my PR? Clone my own branch wget/mattermost-docs branch or somewhat publish a PR against the mattermost/docs repo and link it to this PR (if you proceeded that way, I'm eager to know as well).